src/sidebar/components/markdown-editor.js

```diff
@@ -139,60 +145,73 @@ function Toolbar({ isPreviewing, onCommand, onTogglePreview }) {
     >
       <ToolbarButton
         disabled={isPreviewing}
-        icon="format-bold"
+        iconName="format-bold"
         onClick={() => onCommand('bold')}
         shortcutKey={SHORTCUT_KEYS.bold}
         title="Bold"
       />
       <ToolbarButton
         disabled={isPreviewing}
-        icon="format-italic"
+        iconName="format-italic"
         onClick={() => onCommand('italic')}
         shortcutKey={SHORTCUT_KEYS.italic}
         title="Italic"
       />
       <ToolbarButton
         disabled={isPreviewing}
-        icon="format-quote"
+        iconName="format-quote"
         onClick={() => onCommand('quote')}
         shortcutKey={SHORTCUT_KEYS.quote}
         title="Quote"
       />
       <ToolbarButton
         disabled={isPreviewing}
-        icon="link"
+        iconName="link"
         onClick={() => onCommand('link')}
         shortcutKey={SHORTCUT_KEYS.link}
         title="Insert link"
       />
       <ToolbarButton
         disabled={isPreviewing}
-        icon="insert-photo"
+        iconName="image"
         onClick={() => onCommand('image')}
         shortcutKey={SHORTCUT_KEYS.image}
         title="Insert image"
       />
       <ToolbarButton
         disabled={isPreviewing}
-        icon="functions"
+        iconName="format-functions"
         onClick={() => onCommand('math')}
         title="Insert math (LaTeX is supported)"
       />
       <ToolbarButton
         disabled={isPreviewing}
-        icon="format-list-numbered"
+        iconName="format-list-numbered"
         onClick={() => onCommand('numlist')}
         shortcutKey={SHORTCUT_KEYS.numlist}
         title="Numbered list"
       />
       <ToolbarButton
         disabled={isPreviewing}
-        icon="format-list-bulleted"
+        iconName="format-list-unordered"
         onClick={() => onCommand('list')}
         shortcutKey={SHORTCUT_KEYS.list}
         title="Bulleted list"
       />
       <span className="u-stretch" />
+      <div className="markdown-editor__toolbar-help-link">
+        <a
+          href="https://web.hypothes.is/help/formatting-annotations-with-markdown/"
+          target="_blank"
+          rel="noopener noreferrer"
+          className="markdown-editor__toolbar-button"
```

Comment:
This link could use a title/accessible label.

Reply:
Great catch—done!